specification/edge/Microsoft.Edge.DisconnectedOperations.Management/images.tsp

```diff
@@ -0,0 +1,55 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+import "./disconnectedOperations.tsp";
+import "../Microsoft.Edge.Shared/common.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+using TypeSpec.Versioning;
+
+namespace Microsoft.Edge;
+@doc("Disconnected operations image resource.")
```

Comment:
Consider having a longer description for people who haven't already heard of these